practices/collectiveCode/readme.md

```diff
@@ -0,0 +1,23 @@
+# Collective Code Ownership
+
+In collective code ownership, the entire team is responsible for the code. Everyone works together to produce a product of quality. No one individual is greater than the rest of the team members. This practice helps to reduce the need for hero work and eliminate silos of information within the team.
+
+## The What and the Why
+
+### The What
+
+Code ownership refers to who is allowed to modify and write certain portions of the code. Collective code ownership means the whole team has the ability and even duty to review and edit every piece of code.
```

Comment:
'review' has some baggage in this context that doesn't really apply, 'understand' might be a better fit

Reply:
fixed!